hamilton/driver.py

```diff
@@ -391,6 +391,7 @@ def __init__(
         adapter: Optional[
             Union[lifecycle_base.LifecycleAdapter, List[lifecycle_base.LifecycleAdapter]]
         ] = None,
+        allow_module_overrides: bool = False,
```

Comment:
This is good to have here for the old (non-builder) API/passing through -- add to docstring?